crates/interpreter/src/valid.rs

```diff
@@ -979,3 +976,12 @@ impl<'a, 'm> Expr<'a, 'm> {
         }
     }
 }
+
+fn offset_parser(branch: SideTableBranch, off: isize) -> &[u8] {
```

Comment:
This function should be shared. It could either be in `module.rs`, `parser.rs`, or a new `util.rs`.